text/0156-columnar-encoding.md

```diff
@@ -0,0 +1,2195 @@
+
+> Open question: `aggregation_temporality` and `is_monotonic` are in this proposal represented as column metadata, but
+> they could also be represented as 2 distinct columns if needed. Feedback from the community on this choice will be
+> appreciated.
+
+#### Logs Payload
+
+Although simpler, a logs 'OtlpArrowPayload' takes a similar approach.
+
+| Column name                  | Column type | Required | Description                                                                                                     |
+|------------------------------|-------------|----------|-----------------------------------------------------------------------------------------------------------------|
+| `resource`                   | `struct`    | No       | Structure regrouping the fields of the resource                                                                 |
+| __`attributes`               | `struct`    | No       | Structure regrouping the fields of the resource attributes                                                      |
+| ____`[key]`                  | `dynamic`   | No       | A set of attributes, see [attribute section](#labelattribute-representation) for more details                   |
+| __`dropped_attributes_count` | `uint32`    | No       | The number of resource dropped attributes.                                                                      |
+| __`schema_url`               | `string`    | No       | Schema url.                                                                                                     |
+| `instrumentation_library`    | `struct`    | No       | Structure regrouping the fields of the instrumentation library                                                  |
+| __`name`                     | `string`    | No       | Name of the instrumentation library                                                                             |
+| __`version`                  | `string`    | No       | Version of the instrumentation library                                                                          |
+| `time_unix_nano`             | `uint64`    | Yes      | The time when the event occurred.                                                                               |
+| `severity_number`            | `uint8`     | Yes      | The severity number.                                                                                            |
+| `severity_text`              | `string`    | No       | The severity test.                                                                                              |
+| `name`                       | `string`    | No       | Short event identifier that does not contain varying parts.                                                     |
+| `body`                       | `dynamic`   | No       | The body of the log record (see below for more details).                                                        |
```

Comment:
I'm not sure where to look below for more details, any way to link to the paragraph?

Reply:
Sure, I will add a link.

Reply:
Changed the Arrow Schema representation to be more readable.